priv/templates/talon.gen.theme/admin_lte/templates/layout/sidebar.html.slim

```diff
@@ -1,11 +1,19 @@
 aside.main-sidebar
   section.sidebar style="height: auto;"
     ul.sidebar-menu
+      li.header.text-uppercase Dashboard
+      / TODO: use talon_context to get page/dashboard_paths (DJS)
+      - resource_or_page = @talon_resource || @talon_page
+      = if resource_or_page do
+        = for dashboard_path <- resource_or_page.dashboard_paths(@conn.assigns.talon) do
```

Comment:
Ah, I see.  I was thinking a different order as well as semi-first class treatment for dashboards (see my comments above).  Also, that feels a little flat for more complex applications and not easily scalable to handle grouping. Perhaps my suggestion of page groups could be used here for flexibility? Resource groups could also make sense.